test-e2e/type/ClickOptions.ts

```diff
@@ -0,0 +1,17 @@
+interface ClickOptions {

```

Comment:
You should be able to use the types from playwright.

Reply:
it seems that they did not expose it, this why I create an interface

Reply:
`type ClickOptions = Parameters<Page['click']>[1];`

Reply:
Thanks  @targos